packages/components/src/spinner/stories/index.tsx

```diff
@@ -16,7 +17,9 @@ const sizes = {
 	large: space( 40 ),
 };
 
-export default {
+const meta: Meta<
+	typeof Spinner & { size: 'default' | 'medium' | 'large' }
+> = {
```

Comment:
Let's make a few changes to how this story is written:

- we need to add a snippet of code to the `meta` objects to setup controls and show the docs tab by default
- while we're at is, we could actually remove the extra `size` control (as it's not a prop on the `Spinner` component), and instead just create a separate story that illustrates how the `Spinner` component can be resized to any size, while the stroke width stays unaffected

<details>

<summary>Sample diff of the changes</summary>

```diff
diff --git a/packages/components/src/spinner/stories/index.tsx b/packages/components/src/spinner/stories/index.tsx
index e4f8d9c026..2ffbf680d8 100644
--- a/packages/components/src/spinner/stories/index.tsx
+++ b/packages/components/src/spinner/stories/index.tsx
@@ -1,51 +1,34 @@
 /**
  * External dependencies
  */
-import { css } from '@emotion/react';
 import type { Meta, Story } from '@storybook/react';
 
 /**
  * Internal dependencies
  */
-import { useCx } from '../../utils';
 import { space } from '../../ui/utils/space';
 import Spinner from '../';
 
-const sizes = {
-	default: undefined,
-	medium: space( 20 ),
-	large: space( 40 ),
-};
-
-const meta: Meta<
-	typeof Spinner & { size: 'default' | 'medium' | 'large' }
-> = {
-	title: 'Components/Spinner',
+const meta: Meta< typeof Spinner > = {
 	component: Spinner,
-	argTypes: {
-		size: {
-			options: Object.keys( sizes ),
-			mapping: sizes,
-			control: { type: 'select' },
-		},
+	title: 'Components/Spinner',
+	parameters: {
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
 	},
 };
 export default meta;
 
-const Template: Story<
-	typeof Spinner & { size: 'default' | 'medium' | 'large' }
-> = ( { size } ) => {
-	const cx = useCx();
-	const spinnerClassname = cx( css`
-		width: ${ size };
-		height: ${ size };
-	` );
-	return <Spinner className={ spinnerClassname } />;
-};
+const Template: Story< typeof Spinner > = ( args ) => <Spinner { ...args } />;
+
+export const Default: Story< typeof Spinner > = Template.bind( {} );
+Default.args = {};
 
-export const Default: Story<
-	typeof Spinner & { size: 'default' | 'medium' | 'large' }
-> = Template.bind( {} );
-Default.args = {
-	size: 'default',
+// The width of the Spinner's border is not affected by the overall component's dimensions.
+export const CustomSize: Story< typeof Spinner > = Template.bind( {} );
+CustomSize.args = {
+	style: {
+		width: space( 20 ),
+		height: space( 20 ),
+	},
 };
```
</details>

Reply:
This seems much better thanks for the tip!